packages/gatsby-tinacms-mdx/src/MdxForm.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import { FormOptions, Form } from 'tinacms'
+import { MdxNode } from './mdx-node'
+import { useLocalMdx } from './useMdxForm'
+
+interface MdxFormProps extends Partial<FormOptions<any>> {
+  _mdx: MdxNode
+  render(renderProps: { form: Form; mdx: any }): JSX.Element
+  timeout?: number
+}
+
+export function MdxForm({ _mdx, render, ...options }: MdxFormProps) {
```

Comment:
You can name variables during destructuring. This will let the `MdxFormProps` accept `mdx` instead of `_mdx`.

```suggestion
export function MdxForm({ mdx: _mdx, render, ...options }: MdxFormProps) {
```

Reply:
Right, that was I was thinking too but then we'll have a name conflict on line 30, unless we change that as well.

Reply:
In `RemarkForm.tsx`, they use `remark` and `markdownRemark`.

I couldn't think of another name hence why I used `mdx` and `_mdx`

Reply:
Sorry, I was unclear. The whole component would look like this:

```ts
interface MdxFormProps extends Partial<FormOptions<any>> {
  mdx: MdxNode
  render(renderProps: { form: Form; mdx: any }): JSX.Element
  timeout?: number
}

export function MdxForm({ mdx: _mdx, render, ...options }: MdxFormProps) {
  const [mdx, form] = useLocalMdx(_mdx, options)

  return render({ form: form as Form, mdx })
}
```

The `mdx: _mdx` lets you pass it in as `mdx` and then use it as `_mdx`